api-reference/10 UI Components/dxForm/1 Configuration/alignItemLabels.md

```diff
@@ -5,7 +5,7 @@ default: true
 ---
 ---
 ##### shortDescription
-Specifies whether all item labels are aligned. Applies only if the [labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"*.
+Specifies whether all item labels are aligned. Applies only if the [labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"* or [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the labels.
```

Comment:
"if ... [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the labels" means "if labels are visible", which is kind of an unnecessary thing to say. It's obvious that if labels are hidden, they cannot be aligned.

Reply:
After implementing floating labels, there are 2 types of labels: form.labels and editors.labels. So, customers may confuse may initial variant. May be it's better to specify, that this option affects only form.labels, not editor.labels?

Is the variant below more clear?
"Applies only if the labels are rendered as the form labels, not editor labels ([labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"* or [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the labels)."